app/views/GovukWrapper.scala.html

```diff
@@ -21,7 +21,6 @@
 @import uk.gov.hmrc.play.views.html.layouts.{Head, HeaderNav, Footer, ServiceInfo, MainContentHeader, MainContent, Sidebar, Article, FooterLinks}
```

Comment:
Can you remove the unused import `HeaderNames`.
Can you rename the javascript file `vat-corr-v1` and update the reference to it in `scriptElement` in this file.